keyless/pepper/service/src/jwk.rs

```diff
@@ -16,6 +18,29 @@ use tokio::time::Instant;
 pub static DECODING_KEY_CACHE: Lazy<DashMap<Issuer, DashMap<KeyID, Arc<RSA_JWK>>>> =
     Lazy::new(DashMap::new);
 
+pub async fn get_federated_jwk(jwt: &str) -> Result<Arc<RSA_JWK>> {
+    let payload = parse(jwt)?;
+
+    let re = Regex::new(r"^https://[a-zA-Z0-9-]+\.us\.auth0\.com/$").unwrap();
+
+    if !re.is_match(&payload.claims.iss) {
+        println!("{}", payload.claims.iss);
```

Comment:
use `info!` instead of `println!` and say more than just the iss?

Reply:
left it in by accident